src/Components/Publishing/Layouts/Components/CanvasFooter.tsx

```diff
@@ -18,6 +19,9 @@ export interface CanvasFooterProps {
 
 export const CanvasFooter: React.SFC<CanvasFooterProps> = props => {
   const { article, display, relatedArticles, renderTime } = props
+  const showCollectionRails = ["standard", "feature", "news"].includes(
```

Comment:
This can be updated to use the `showCollectionsRail` prop from force, my pr is now updated to include your suggestion to limiting layouts from the top down. https://github.com/artsy/force/pull/3589